RELEASES.md

```diff
@@ -0,0 +1,106 @@
+# APM Server release checklist
+
+## Before the release
+
+* Create a PR that:
+    1. Uncomments out relevant release notes. Release notes are in the [changelogs](https://github.com/elastic/apm-server/tree/master/changelogs).
+    2. Updates the `:stack-version:`
+    * [Sample PR](https://github.com/elastic/apm-server/pull/2064/files)
+
+* Update `vendor/github.com/elastic/beats/libbeat/version/version.go`
+    * [Sample PR](https://github.com/elastic/apm-server/pull/1886)
+
+* The following may also need to be updated manually:
+    * APM Overview's [release highlights](https://github.com/elastic/apm-server/blob/master/docs/guide/apm-release-notes.asciidoc) - Anything exciting across the APM stack!
+    * APM Overview's [breaking changes](https://github.com/elastic/apm-server/blob/master/docs/guide/apm-breaking-changes.asciidoc) - Any breaking changes across the APM stack.
+    * APM Server's [breaking changes](https://github.com/elastic/apm-server/blob/master/docs/breaking-changes.asciidoc) - Any APM Server breaking changes.
+    * APM Server's [upgrade guide](https://github.com/elastic/apm-server/blob/master/docs/upgrading.asciidoc).
+
+* Script for determining if changelogs are synced: https://gist.github.com/graphaelli/92fd6b5ff08d69600e880df2a234ac35
```

Comment:
feel free to check this into the this repo as part of this change

Reply:
Ah, yeah, much better idea.